### PR TITLE
DOCS-2208: Create SCALE Replication.md

### PR DESCRIPTION
**Pull request by BonnieF**
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

**Comment by Mrt134** [comment]
Everything appears to build and render ok - pinging @themylogin for a quick technical review

File: content/SCALE/DataProtection/Replication.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+title: "Replication"
+weight: 100
+---
+
+## Remote Replication
+
+
+{{< toc >}}
+
+Configure SSH in TrueNAS before creating a remote replication task. This ensures that both systems can connect to each other and new snapshots are regularly available for replication.
```
Comment (themylogin) [inline content/SCALE/DataProtection/Replication.md:11]: We don't require both systems to be able to connect to each other. Local system connects to the remote system, not the other way around.

File: content/SCALE/DataProtection/Replication.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+
+TrueNAS shows how many snapshots are available for replication.
+We recommend you manually snapshot the sources or create a periodic snapshot task *before* creating the replication task.
+However, when the sources are on the local system and don't have any existing snapshots, TrueNAS can create a basic periodic snapshot task and snapshot the sources immediately before starting the replication. Enabling *Recursive* replicates all snapshots contained within the selected source dataset snapshots.
+
+![TasksReplicationTasksAddSourceSCALE](/images/SCALE/RepAddSourceSCALE.png "Choosing a Local Source")
+
+Local sources can also use a naming schema to identify any custom snapshots to include in the replication.
+Remote sources require entering a *snapshot naming schema* to identify the snapshots to replicate.
+A naming schema is a collection of [strftime](https://www.freebsd.org/cgi/man.cgi?query=strftime) time and date strings and any identifiers that a user might have added to the snapshot name.
```
Comment (themylogin) [inline content/SCALE/DataProtection/Replication.md:62]: We probably need an example here.

There also is a fourth option: replicating snapshots with names matching a specific regex, https://github.com/truenas/zettarepl Ctrl-F, `name-regex`

File: content/SCALE/DataProtection/Replication.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+{{< /hint >}}
+
+{{< /tab >}}
+{{< tab "Security and Task Name" >}}
+{{< hint info >}}
+Using encryption for SSH transfer security is always recommended.
+{{< /hint >}}
+
+In situations where two systems within an absolutely secure network are used for replication, disabling encryption speeds up the transfer.
+However, the data is completely unprotected from malicious sources.
```
Comment (themylogin) [inline content/SCALE/DataProtection/Replication.md:89]: Unprotected from eavesdropping

File: content/SCALE/DataProtection/Replication.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+  * Choose sources for snapshot replication.
+    * Remote sources require an SSH connection.
+    * TrueNAS shows how many snapshots will be replicated.
+  * Define the snapshot destination.
+    * A remote destination requires an SSH connection.
+    * Choose destination or define manually by typing a path.
+      * Adding a new name on the end of the path creates a new dataset.
+  * Choose replication security.
+    * We always recommend Replication with encryption.
+    * Disabling encryption is only meant for absolutely secure networks.
```
Comment (themylogin) [inline content/SCALE/DataProtection/Replication.md:28]: If we are talking about zfs dataset encryption then it's probably "Disabling encryption is only meant for absolutely secure and trusted destinations."

File: content/SCALE/DataProtection/Replication.md [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+{{< /tab >}}
+{{< tab "Security and Task Name" >}}
+{{< hint info >}}
+Using encryption for SSH transfer security is always recommended.
+{{< /hint >}}
+
+In situations where two systems within an absolutely secure network are used for replication, disabling encryption speeds up the transfer.
+However, the data is completely unprotected from malicious sources.
+
+Choosing no encryption for the task is the same as choosing the *SSH+NETCAT* transport method from the advanced options screen.
```
Comment (themylogin) [inline content/SCALE/DataProtection/Replication.md:91]: No, it's not, SSH+NETCAT is even faster (much faster, but it does not support any encryption)

**Review by themylogin — CHANGES_REQUESTED** [review]

File: content/SCALE/DataProtection/Replication.md (hunk 1 above)
Comment (BonnieF) [reply content/SCALE/DataProtection/Replication.md]: fixed

File: content/SCALE/DataProtection/Replication.md (hunk 2 above)
Comment (Mrt134) [reply content/SCALE/DataProtection/Replication.md:62]: @BonnieF you can see examples of naming schemas by creating a periodic snapshot task then editing it and finding the value. A basic naming schema that is created with a generic replication task is `auto-%Y-%m-%d_%H-%M`
@themylogin is the regex option something that is configurable from the UI?

**Comment by Mrt134** [comment]
Updated https://jira.ixsystems.com/browse/DOCS-2396 to include updating for regex selection and naming schema examples. Going to do a final check and merge on this one so @BonnieF can continue working on her other branch.

**Review by Mrt134 — APPROVED** [review]

**Pull request merged**